notebooks/DrizzlePac/using_updated_astrometry_solutions/using_updated_astrometry_solutions.ipynb

```diff
@@ -0,0 +1,753 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using updated astrometry solutions"
```

Comment:
I'm finding this title a bit hard to fully understand the context.  Perhaps "Choosing alternate astrometry solutions for drizzling"?

(If you make the change here, it's probably good to also rename the file and directory)

Reply:
The context will usually be for drizzling, but not always so I think it might be best to keep it general. I'll brainstorm a title that makes more sense and gives more context.

Reply:
If it's ok with you I think i'll just leave this title for now, I can't think of anything better

Reply:
The idea was to keep it general, as I didn't want users to think this was something to ignore if they're not necessarily running drizzle.  It's really applicable for any case where the WCS is used, as these are yielding different WCS's.

Reply:
Alright, point taken.  Then we can leave it as is.